setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name='mlflow-databricks-artifacts',
+    version='1.0.0',
+    description='Plugin to create and access MLflow-managed artifacts on Databricks',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author='Databricks',
+    packages=find_packages(exclude=["tests", "tests.*"]),
+    install_requires=[
+        'mlflow>=1.0.0',
+    ],
+    entry_points={
+        "mlflow.artifact_repository": [
+            "dbfs=mlflow_databricks_artifacts.store.entrypoint:dbfs_artifact_repo_factory",
```

Comment:
In accordance with the instructions for registering third-party artifact repository plugins with MLflow (https://mlflow.org/docs/latest/plugins.html#defining-a-plugin), we register the `dbfs` URI scheme to map to this plugin's artifact-ACL-aware `mlflow_databricks_artifacts.store.entrypoint#dbfs_artifact_repo_factory` function